plugin/generator.go

```diff
@@ -19,26 +19,41 @@ import (
 	"github.com/docker/docker/client"
 )
 
-var proxyServiceTasks bool
-var caddyNetworks map[string]bool
+var labelPrefixFlag string
+var labelPrefixEnv string
+var proxyServiceTasksFlag bool
+var proxyServiceTasksEnv string
+var isTrue = regexp.MustCompile("(?i)^(true|yes|1)$")
 
 func init() {
-	flag.BoolVar(&proxyServiceTasks, "proxy-service-tasks", false, "Proxy to service tasks instead of VIP")
-}
+	flag.StringVar(&labelPrefixFlag, "docker-label-prefix", "caddy", "Prefix for Docker labels")
+	flag.BoolVar(&proxyServiceTasksFlag, "docker-proxy-service-tasks", false, "Proxy to service tasks instead of VIP")
```

Comment:
This is a breaking change, we should continue using `proxy-service-tasks`

Reply:
What about to have both for now and remove the old one later? Give people chance to change it? Just mention it in the changelog that it will be removed in next release.